packages/dokeysto_lz4/dokeysto_lz4.3.0.2/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+name: "dokeysto_lz4"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/dokeysto"
+bug-reports: "https://github.com/UnixJunkie/dokeysto/issues"
+dev-repo: "git+https://github.com/UnixJunkie/dokeysto.git"
+license: "LGPL-2.1"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test_lz4"] {with-test}
+]
+depends: [
+  "ocaml"
+  "dune" {>= "1.0" & < "2.0"}
```

Comment:
```suggestion
  "dune" {>= "1.11"}
```
According to your `dune-project` file, this package requires at least dune 1.11, also this package should now be compatible with dune 2.0